lib/nexus/tests/nexus_plot_spe1.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+   Copyright (C) 2017 Statoil ASA, Norway.
+
+   The file 'nexus_plot_load.cpp' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <algorithm>
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <vector>
+
+
+#include <ert/util/test_util.hpp>
+#include <ert/nexus/nexus_plot.hpp>
+
+using namespace nex;
+
+
+namespace {
+    const std::vector< std::string > spe1_classnames {{
+        "WELL    ", "NODE    ", "CONN    ", "REGION  ", "FIELD   ", "CONNLIST"
```

Comment:
These spaces are on the way out in a subsequent PR?

Reply:
I'll fix this before the merge.